apps/site/app/[locale]/next-data/og/[category]/[title]/route.tsx

```diff
@@ -4,32 +4,34 @@ import { ImageResponse } from 'next/og';
 
 import { DEFAULT_CATEGORY_OG_TYPE } from '#site/next.constants.mjs';
 import { defaultLocale } from '#site/next.locales.mjs';
-import { hexToRGBA } from '#site/util/hexToRGBA';
 
 // TODO: use CSS variables instead of absolute values
 const CATEGORY_TO_THEME_COLOUR_MAP = {
-  announcement: '#1a3f1d',
-  release: '#0c7bb3',
-  vulnerability: '#ae5f00',
+  announcement: 'rgb(26, 63, 29)',
+  release: 'rgb(12, 123, 179)',
+  vulnerability: 'rgb(174, 95, 0)',
 };
 
-type Category = keyof typeof CATEGORY_TO_THEME_COLOUR_MAP;
-
-type DynamicStaticPaths = { locale: string; category: Category; title: string };
-type StaticParams = { params: Promise<DynamicStaticPaths> };
+type StaticParams = {
+  params: Promise<{
+    locale: string;
+    category: keyof typeof CATEGORY_TO_THEME_COLOUR_MAP;
+    title: string;
+  }>;
+};
 
 // This is the Route Handler for the `GET` method which handles the request
 // for generating OpenGraph images for Blog Posts and Pages
 // @see https://nextjs.org/docs/app/building-your-application/routing/router-handlers
 export const GET = async (_: Request, props: StaticParams) => {
   const params = await props.params;
 
-  const categoryColour =
```

Comment:
drive by nit: i've never seen us have an authoritative stance on English or American spellings, and as such allow both personally

Reply:
A futher drive-by opinion: given in CSS it always uses the American spelling, I think the same should be done in other contexts for consistency (much as I would like the world to use the correct spelling /jk).

Reply:
I'm also +1 on American spelling, but at the current point in time, it's only bike shedding, so perhaps a follow up enforcing consistent spelling.